app/src/main/res/values-en/strings.xml

```diff
@@ -123,6 +123,7 @@
     <string name="home_more_terms">Terms and Conditions of Use</string>
     <string name="home_more_title">More</string>
     <string name="home_startSetup">Set up online ID card</string>
+    <string name="identification_attributeConsent_header">Grant release for »%1s«.</string>
```

Comment:
I would suggest using the prefix `_title` instead of `_header` to be more consistent with the other keys.

Reply:
Yes good point!